classes/controller/AdminController.php

```diff
@@ -590,7 +590,7 @@ public function initMultistoreHeader(): void
         }
 
         $this->context->smarty->assign([
-            'multistore_header' => $this->container->get('prestashop.core.admin.multistore')->header($this->lockedToAllShopContext)->getContent(),
+            'multistore_header' => $this->get('prestashop.core.admin.multistore')->header($this->lockedToAllShopContext)->getContent(),
```

Comment:
This is possible 😮 ? You can use `get()` in AdminController to retrieve directly services from the container?

Reply:
seems like it, see in `Controller.php`

```php
public function get($serviceId)
{
    return $this->container->get($serviceId);
}
```